[2025-07-05T08:38:07.756+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-07-05T08:38:07.778+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: lakehouse_pipeline.bronze_to_silver manual__2025-07-05T08:38:05.813493+00:00 [queued]>
[2025-07-05T08:38:07.784+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: lakehouse_pipeline.bronze_to_silver manual__2025-07-05T08:38:05.813493+00:00 [queued]>
[2025-07-05T08:38:07.784+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-07-05T08:38:07.792+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): bronze_to_silver> on 2025-07-05 08:38:05.813493+00:00
[2025-07-05T08:38:07.797+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'lakehouse_pipeline', 'bronze_to_silver', 'manual__2025-07-05T08:38:05.813493+00:00', '--job-id', '2', '--raw', '--subdir', 'DAGS_FOLDER/pipeline_dag.py', '--cfg-path', '/tmp/tmp19m8curm']
[2025-07-05T08:38:07.799+0000] {standard_task_runner.py:91} INFO - Job 2: Subtask bronze_to_silver
[2025-07-05T08:38:07.800+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=202) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-07-05T08:38:07.800+0000] {standard_task_runner.py:63} INFO - Started process 203 to run task
[2025-07-05T08:38:07.830+0000] {task_command.py:426} INFO - Running <TaskInstance: lakehouse_pipeline.bronze_to_silver manual__2025-07-05T08:38:05.813493+00:00 [running]> on host be55293f307d
[2025-07-05T08:38:07.886+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='lakehouse_pipeline' AIRFLOW_CTX_TASK_ID='bronze_to_silver' AIRFLOW_CTX_EXECUTION_DATE='2025-07-05T08:38:05.813493+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-07-05T08:38:05.813493+00:00'
[2025-07-05T08:38:07.887+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-07-05T08:38:07.911+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-07-05T08:38:07.911+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'spark-submit /app/bronze_to_silver.py']
[2025-07-05T08:38:07.920+0000] {subprocess.py:86} INFO - Output:
[2025-07-05T08:38:07.921+0000] {subprocess.py:93} INFO - /usr/bin/bash: line 1: spark-submit: command not found
[2025-07-05T08:38:07.921+0000] {subprocess.py:97} INFO - Command exited with return code 127
[2025-07-05T08:38:07.922+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-07-05T08:38:07.928+0000] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 400, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/bash.py", line 243, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 127.
[2025-07-05T08:38:07.931+0000] {taskinstance.py:1206} INFO - Marking task as FAILED. dag_id=lakehouse_pipeline, task_id=bronze_to_silver, run_id=manual__2025-07-05T08:38:05.813493+00:00, execution_date=20250705T083805, start_date=20250705T083807, end_date=20250705T083807
[2025-07-05T08:38:07.940+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 2 for task bronze_to_silver (Bash command failed. The command returned a non-zero exit code 127.; 203)
[2025-07-05T08:38:07.974+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2025-07-05T08:38:08.014+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-07-05T08:38:08.018+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
